trade-finance-manager-api/src/v1/controllers/amendment.controller-updateFacilityAmendment.test.js

```diff
@@ -138,4 +152,51 @@ describe('controllers - amendment, function updateFacilityAmendment', () => {
       );
     });
   });
+
+  describe('updateFacilityAmendment', () => {
+    const { res } = createMocks({ params: { amendmentId, facilityId }, user: underwriter, body: requestBody });
+    const auditDetails = { id: undefined, userType: 'tfm' };
+
+    it('should add isReadyForApproval property to ukefDecision object, if no UKEF decision has been added to the amendment', async () => {
```

Comment:
I think this would be nicer for each test, but happy to keep as is if this is the existing pattern. What do you think?

```ts
describe('when ukefDecision exists', () => {
  it('should ...', () => {};
});

describe('when ukefDecision does not exist', () => {
  it('should ...', () => {};
});
```